release/ray_release/glue.py

```diff
@@ -289,6 +291,14 @@ def run_release_test(
             logger.exception(e)
             command_results = {}
 
+        try:
+            command_runner.save_metrics(start_time_unix)
+            metrics = command_runner.fetch_metrics()
+        except Exception as e:
+            logger.error("Could not fetch metrics for test command")
+            logger.exception(e)
```

Comment:
`logger.exception` is just `logger.error` with printing the traceback, so we should probably do this for better readability

```suggestion
            logger.exception(f"Could not fetch metrics for test command: {e}")
```